npc/re/jobs/2e/rebellion.txt

```diff
@@ -0,0 +1,1093 @@
+		if (.@st_job1 > 0) {
+			delitem(Steel_Article, .@st_job1);
+			mes("[Elwin Conick]");
+			mes("Any leftover Steel Artifacts must be returned after the test.");
+			close;
+		}
+		if (.@st_job2 > 0) {
+			delitem(Steel_Article_, .@st_job2);
+			mes("[Elwin Conick]");
+			mes("Any leftover Steel Artifacts must be returned after the test.");
+			close;
+		}
+		if (countitem(Tiny_Flame) > 0) {
+			delitem(Tiny_Flame, 1);
+			mes("[Elwin Conick]");
+			mes("Time over. You failed the test. Let me take all your leftover test materials.");
+			close;
+		}
+		setquest(12342);
+		completequest(12341);
+		.@showDialog = true;
+	} else if (questprogress(12342) == 1) {
+		.@st_job1 = countitem(Steel_Article);
+		.@st_job2 = countitem(Steel_Article_);
+		if (.@st_job1 > 0) {
+			delitem(Steel_Article, .@st_job1);
+			mes("[Elwin Conick]");
+			mes("Any leftover Steel Artifacts must be returned after the test.");
+			close;
+		}
+		if (.@st_job2 > 0) {
+			delitem(Steel_Article_, .@st_job2);
+			mes("[Elwin Conick]");
+			mes("Time over. You failed the test. Let me take all your leftover test materials.");
+			close;
+		}
+		if (countitem(Tiny_Flame) > 0) {
+			delitem(Tiny_Flame, 1);
+			mes("[Elwin Conick]");
+			mes("Time over. You failed the test. Let me take all your leftover test materials.");
+			close2;
+			end;
+		}
+		.@showDialog = true;
+	}
+	if (.@showDialog) {
+		mes("[Elwin Conick]");
+		mes("Listen up. I'm not going to great lengths trying to explain the test to you.");
+		next;
+		select("Okay.");
+		mes("[Elwin Conick]");
+		mes("This is where the firearm molds are cast. Everywhere you can see Steel Artifacts are being produced.");
+		next;
+		mes("[Elwin Conick]");
+		mes("You'll be asked to control switches on the machine and select 5 standard Steel Artifacts.");
+		next;
+		mes("[Elwin Conick]");
+		mes("You'll then take them to the next Assembly Room and assemble them into a complete firearm. You'll be given 3 minutes to finish all that. ^0000ffTo check the elapsed time, you'll be transformed into a different job^000000.");
+		next;
+		mes("[Elwin Conick]");
+		mes("I'll be in the next room, waiting for you to bring me a complete firearm. Remember, ^0000ffyou fail the test when your transformation expires after 3 minutes^000000.");
+		montransform(G_L_SHECIL, 180000);
+		close;
+	}
+	mes("[Elwin Conick]");
+	mes("Ah? Are you lost? You need to register first and then get back here.");
+	close2;
+	warp("job_gun", 81, 29);
+	next;
+end;
+}
+
+job_gun,198,39,3	script	Elwin Conick#3	4_F_REBELLION,2,2,{
+	if (Class != Job_Gunslinger) {
+		mes("[Elwin Conick]");
+		mes("If you don't want to join Rebellions, just get out.");
+		close2;
+		warp("job_gun", 81, 29);
+		end;
+	}
+	if (questprogress(12342) != 1) {
+		mes("[Elwin Conick]");
+		mes("Ah? Are you lost? You need to register first and then get back here.");
+		close2;
+		warp("job_gun", 81, 29);
+		end;
+	}
+	if (getstatus(SC_MONSTER_TRANSFORM, 1) == 2431) {
```

Comment:
```suggestion
	if (getstatus(SC_MONSTER_TRANSFORM, 1) == G_L_SHECIL) {
```

Reply:
done, will be pushed together in next commit